lib/temporal/client.rb

```diff
@@ -372,6 +372,25 @@ def list_closed_workflow_executions(namespace, from, to = Time.now, filter: {})
       fetch_executions(:closed, { namespace: namespace, from: from, to: to }.merge(filter))
     end
 
+    # Fetch a workflow's cron schedule
+    #
+    # @param namespace [String]
+    # @param workflow_id [String]
+    # @param run_id [String] If not specified, the most recent run will be used
+    #
+    # @return [String] the schedule in crontab format, or nil if there is no schedule
+    def get_cron_schedule(namespace, workflow_id, run_id: nil)
```

Comment:
@jeffschoner-stripe could you add unit specs in `spec/unit/lib/temporal/client_spec.rb` too?

Reply:
I've added unit tests and a fabricator for creating the initial history entry with a cron schedule too